### PR TITLE
Added tooltip on hover on the badges of individual member profile page

### LINKED ISSUE #3797: Add tooltip on hover effect on the badges in Individual member profile page.
### Current Behavior

Currently, there is no tooltip on hovering over badges in the individual pages of members on the page https://layer5.io/community/members.
image

### Desired Situation

Add the tooltip while hovering over the badges.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by DarrenDsouza7273**
**Description**
Implemented a tooltip for badges on the community member page

This PR fixes #3797 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c2c3fee407cb07f2790bc0e9aefcb0cab6e68d24 at: https://64992415e3d5790351f94a87--layer5.netlify.app

**Comment by DarrenDsouza7273** [comment]
@vishalvivekm check it outt now

**Comment by vishalvivekm** [comment]
@DarrenDsouza7273 
Service mesh performance badge not visible on individual member's page: 

Screenshot 2023-06-26 143304

**Comment by DarrenDsouza7273** [comment]
Do i add the badge name as "Smp" or service mesh performance? @vishalvivekm

**Comment by l5io** [comment]
🚀 Preview for commit 56e33839aaa31ca3f268e5bc5ab0428c8660bab8 at: https://649962ff3a5864273a82ca93--layer5.netlify.app

**Comment by DarrenDsouza7273** [comment]
> @DarrenDsouza7273 Service mesh performance badge not visible on individual member's page:
> 
> Screenshot 2023-06-26 143304
 Done @vishalvivekm

**Comment by l5io** [comment]
🚀 Preview for commit c5bf950d58a864c6b18ad429c22569189da3bdd2 at: https://649ad6698433e000933b27bd--layer5.netlify.app

File: src/sections/Community/Member-single/memberSingle.style.js [hunk 1]
```diff
@@ -127,8 +135,8 @@ export const MemberSingleWrapper = styled.div`
                     
                     &:hover{
                         box-shadow: 0 1px 1.5px 0 rgba(0,0,0,.12),0 1px 1px 0 rgba(0,0,0,.24);
-                        background: white;
-                        
+                        background :  ${props => props.theme.whiteToGrey737373};
```
[the remark this answers is not in the document]
Comment (DarrenDsouza7273) [reply src/sections/Community/Member-single/memberSingle.style.js:138]: Thanks @8rxn

**Comment by l5io** [comment]
🚀 Preview for commit b2ebb13b2d197786696360c22e6e5dbdd15e067d at: https://64a3a66aac33f71947fd475b--layer5.netlify.app

**Comment by samyakshah3008** [comment]
@DarrenDsouza7273  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by DarrenDsouza7273** [comment]
@samyakshah3008 Not sure if i will be able to attend,my college starts tomorrow

**Comment by DarrenDsouza7273** [comment]
Theres two options 
- Either move the badges to the right such that the whole tooltip is visible
- or a minute shift of the name towards the right 
If there's any other ways this can be resolved please let me know for now i've found these ways to solve the issues @gauravchauhan2403

**Comment by l5io** [comment]
🚀 Preview for commit be7e796710e2921d9491a53b5c3a0c7e912918f5 at: https://64aea5b27f538605100290f8--layer5.netlify.app

**Comment by DarrenDsouza7273** [comment]
please check #4543

**Pull request closed without merging**